2-mandatory/3-stocks.js

```diff
@@ -34,7 +34,17 @@ const CLOSING_PRICES_LAST_5_DAYS_FOR_ALL_STOCKS = [
         Functions can help with this!
 */
 function getAveragePrices(closingPricesForAllStocks) {
-    // TODO
+  // TODO
+  const resultArray = [];
+  for (let i = 0; i < STOCKS.length; i++) {
+    let total = 0;
+    for (let j = 0; j < closingPricesForAllStocks[i].length; j++) {
+      total += closingPricesForAllStocks[i][j];
+    }
+    const average = total / closingPricesForAllStocks[i].length;
+    resultArray.push(parseFloat(average.toFixed(2)));
```

Comment:
Here, you are converting a number into a string and then back to a number. 
I'm nit picking here, but you could use
`Math.round(average * 100) / 100`